server.js

```diff
@@ -3,24 +3,49 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
+import { Thought } from './models/thought'
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughtsMONGOAPI"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// My routes
+app.get('/', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  if (thoughts) {
+    res.status(200).json(thoughts)
+  } else {
+    res.status(400).json({ message: 'Could not find thoughts', error: err.errors })
+  }
+})
+
+// POST thoughts to page
+app.post('/', async (req, res) => {
+    const { message } = req.body
+    try {
+    const thought = await new Thought({ message }).save()
+    res.status(200).json(thought)
+  } catch (err) {
+    res.status(400).json({ message: 'Could not post thought', error: err.error })
+  }
+})
+
+// POST likes to page
+app.post('/:thoughtId/likes', async (req, res) => {
+  const { thoughtId } = req.params
+  try {
+    await Thought.updateOne({'_id': thoughtId }, {$inc: {'likes': 1}})
```

Comment:
The only thing might be that you should remove the ' ' around likes?
`{ $inc: { likes: 1 } }`
other than that, this endpoint seems to be correct.